docs/source/project_documentation/dev_guide/language_api/modeling_inheritance.md

```diff
@@ -10,48 +10,58 @@ Let's take a look at how AaC supports `Schema` inheritance.
+
+```{eval-rst}
+.. literalinclude:: ../../../../../python/features/shapes/shapes.aac
+    :lines: 12-48
+    :language: yaml
 ```
 
+Each of the above schemas have their own properties that help distinguish them as their specific shape. But they all require a perimeter and area field as a valid shape, and so inherit from `Shape`.
+
 ## Inheritance Modifiers
 
-Just as in most programming languages, the AaC language provides some basic modifiers to control what you can and can't do with inherited `schema` definitions.  
+Just as in most programming languages, the AaC language provides some basic modifiers to control what you can and can't do with inherited `schema` definitions.
 
 - `abstract` - This attribute allows declaring a schema as an abstract type. This means it cannot be "instantiated" directly in a definition.  In AaC, this means the schema is either a root type or field in another `Schema`.
 - `final` - This allows declaring a schema as a final type, meaning it cannot be extended by another `Schema`.  In AaC this means the final schema cannot appear in another schema's extends field.
 
-To use these inheritance modifiers you simply add them to the `modifiers` field of the `schema`.  Let's update the previous example to see how this works.
+To use these inheritance modifiers you simply add them to the `modifiers` field of the `schema`.  Let's take another look at the above example to see how it works:
 
 ```{eval-rst}
-.. literalinclude:: ../../../../python/features/docs_examples/dev_guide_language_api_modeling_inheritance/docs_inhertaince_two.aac
+.. literalinclude:: ../../../../../python/features/shapes/shapes.aac
+    :lines: 1-10
+    :emphasize-lines: 4-5
     :language: yaml
-    :lines: 1-22
-    :emphasize-lines: 4-5, 15-16
 ```
 
-We can no longer instantiate a `Person` because it is now `abstract`.  We would also be prevented from creating something like `SalaryEmployee` that would extend `Employee` because `Employee` has the `final` modifier.
+As you can see, the `Shape` schema has an `abstract` modifier.  This means that `Shape` can no longer be instantiated on its own.  To make a shape, you must create or use a schema which inherits from `Shape`.
+
+```{eval-rst}
+.. literalinclude:: ../../../../../python/features/shapes/shapes.aac
+    :lines: 51-64
+    :emphasize-lines: 6-7
+    :language: yaml
+```
+
+The above `square` schema inherits from Shape as well, but it has a `final` modifier.  This means that no other schema can inherit from Square.
 
 ### Using Inheritance
 
 Now that we can define inheritance, how can we use it.  The intended use case is to allow polymorphism in your field declaration.  Let's look at another example to explain.
 
 ```{eval-rst}
-	.. literalinclude:: ../../../../../python/features/docs_examples/dev_guide_language_api_modeling_inheritance/docs_inheritance_two.aac
-	    :language: yaml
-	```
+.. literalinclude:: ../../../../../python/features/shapes/shapes.aac
```

Comment:
I would put a highlight within this example for the `shapes` field entry since it is what is being discussed in the following paragraphs.